sdk/core/azure-core-amqp/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 # Release History
 
-## 2.2.0-beta.2 (Unreleased)
+## 2.0.5 (2021-05-07)
```

Comment:
@alzimmermsft did we intentionally jump back to 2.0.5 for amqp package? It looks like we have been jumping around in our versions for this package and it would be good to understand why.

Reply:
I believe @hemanttanwar has a better understanding on this. As far as I know `azure-core-amqp` has been shipping betas out of a feature branch as it would be disruptive/incompatible with the guidance of shipping hotfixes/patch GAs out of a release branch.